userspace/libpman/src/stats.c

```diff
@@ -234,17 +238,57 @@ struct metrics_v2 *pman_get_metrics_v2(uint32_t flags, uint32_t *nstats, int32_t
 			g_state.stats[MODERN_BPF_N_DROPS_SCRATCH_MAP].value.u64 += cnt_map.n_drops_max_event_size;
 			g_state.stats[MODERN_BPF_N_DROPS].value.u64 += (cnt_map.n_drops_buffer + cnt_map.n_drops_max_event_size);
 
+			if((flags & METRICS_V2_KERNEL_COUNTERS_PER_CPU))
```

Comment:
we could also remove it and accept the cost of looping between all the CPU 2 times if both `METRICS_V2_KERNEL_COUNTERS_PER_CPU` and `METRICS_V2_KERNEL_COUNTERS` are enabled